src/components/widgets/Mo2PercentWidget.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import {IPoint} from 'types/IPoint';
+import {BarChart} from "components/widgets/shared/BasicBarGraph";
+import {XYPlotTemplate} from 'components/widgets/shared/XYPlotTemplate';
+import {
+  BasicHorizontalAxis,
+  BasicVerticalAxis
+} from 'components/widgets/shared/BasicAxes';
+
+const MIN_POINTS_BEFORE_SHOW = 6;
```

Comment:
This might take a long time to show with 6 points

Reply:
Changed min-value to 3, 2 looked a little bit too odd